Signal/src/ViewControllers/ConversationView/ConversationViewItem.m

```diff
@@ -260,12 +260,18 @@ - (nullable ConversationViewCell *)measurementCell
     return measurementCell;
 }
 
-- (CGFloat)vSpacingWithLastLayoutItem:(id<ConversationViewLayoutItem>)lastLayoutItem
+- (CGFloat)vSpacingWithLastLayoutItem:(ConversationViewItem *)lastItem
```

Comment:
I think you got this in the last, erm.. previous PR, but I prefer to use `previous` over `last` whenever possible to distinguish from "final"

Reply:
Yup, already done.